src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,31 @@
+
+body,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
```

Comment:
do not mix resetting and normalizing in one file

Reply:
sure